### PR TITLE
Rocky release

### PR DESCRIPTION
**Pull request by ramereth**

### CODE REVIEW

**Comment by ramereth** [comment]
[OpenStack Rocky Release](https://trello.com/c/rM5J3inr/1299-openstack-rocky-release)

File: files/default/upgrade-controller.sh [hunk 1]
```diff
@@ -25,8 +20,8 @@ yum -d1 -y upgrade \*keystone\* python2-oslo-config
 yum -d1 -y upgrade \*horizon\*
 keystone-manage token_flush
 su -s /bin/sh -c "keystone-manage db_sync" keystone
-keystone-manage fernet_setup --keystone-user keystone --keystone-group keystone
-keystone-manage credential_setup --keystone-user keystone --keystone-group keystone
+#keystone-manage fernet_setup --keystone-user keystone --keystone-group keystone
+#keystone-manage credential_setup --keystone-user keystone --keystone-group keystone
```
Comment (khashf) [inline files/default/upgrade-controller.sh]: Why didn't you remove these lines instead of commenting them out?

**Comment by ramereth** [comment]
!bump major

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v8.0.0.  Have a nice day!

**Pull request merged**